pnpm-lock.yaml

```diff
@@ -34524,7 +34505,7 @@ packages:
     dev: false
 
   github.com/Jack-Works/webpack/528c91e564d5756e21c9c462b607d913452af770(@swc/core@1.3.70)(webpack-cli@5.1.4):
-    resolution: {tarball: https://codeload.github.com/Jack-Works/webpack/tar.gz/528c91e564d5756e21c9c462b607d913452af770}
+    resolution: {commit: 528c91e564d5756e21c9c462b607d913452af770, repo: git+ssh://git@github.com/Jack-Works/webpack.git, type: git}
```

Comment:
Please remove these two resolution changes. They will crash the `pnpm install` in GitHub Actions.